Business/Concrete/CategoryManager.cs

```diff
@@ -1,27 +1,72 @@
-﻿using WebNetSample.Business.Abstract;
+﻿using WebNetSample.Entity.Dtos;
+using WebNetSample.Business.Abstract;
 using WebNetSample.DataAccess.Abstract;
 using WebNetSample.Entity.Concrete;
+using AutoMapper;
 
 namespace WebNetSample.Business.Concrete;
 
 public class CategoryManager : ICategoryService
 {
-    private ICategoryRepository _categoryRepository;
+    private readonly ICategoryRepository _categoryRepository;
+    private readonly IMapper _mapper;
 
-    public CategoryManager(ICategoryRepository categoryRepository)
+    public CategoryManager(ICategoryRepository categoryRepository,
+        IMapper mapper)
     {
         _categoryRepository = categoryRepository;
+        _mapper = mapper;
     }
 
-    public async Task<Category> GetByIdAsync(Guid categoryId) => 
-        await _categoryRepository.GetAsync(entity => entity.Id == categoryId);
+    public async Task<CategoryDetailDto> GetByIdAsync(Guid categoryId)
+    {
+        var categoryInfo = await _categoryRepository.GetAsync(entity => entity.Id == categoryId);
+
+        var mappedCategory = _mapper.Map<CategoryDetailDto>(categoryInfo);
+
+        return mappedCategory;
+    } 
+
+    public async Task<List<CategoryDetailDto>> GetListAsync()
```

Comment:
From business point of view `GetList`, honestly, tells nothing and might be confusing. Let's replace it with `GetAllAsync`